core/git_command.py

```diff
@@ -65,15 +65,15 @@ def __init__(self, process):
 
     def read_stdout(self):
         try:
-            for line in self.process.stdout:
+            for line in iter(self.process.stdout.readline, b""):
```

Comment:
Can you explain what you do here?

Reply:
The original code prints the buffer by bytes and I changed it to print the buffer by lines as what it is supposed to do

Reply:
ok, What are the arguments to iter?

Reply:
https://docs.python.org/3/library/functions.html#iter

Reply:
So this will read until it gets an empty line?

Reply:
No. It will read until the end of the buffer. `readline` will return `b"\n"` for an empty line.

Reply:
ok,